File: class11/java/AP1110-CalculoSalarioLiquido.java
import java.util.Scanner;
public class CalculoSalarioLiquido {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Horas trabalhadas: ");
        double horas = sc.nextDouble();
        System.out.print("Valor por hora: R$ ");
        double valorHora = sc.nextDouble();
        double bruto = horas * valorHora;
        double inss = bruto * 0.08;
        double liquido = bruto - inss;
        System.out.printf("Salário Bruto:   R$ %.2f%n", bruto);
        System.out.printf("Desconto INSS:   R$ %.2f%n", inss);
        System.out.printf("Salário Líquido: R$ %.2f%n", liquido);
        sc.close();
    }
}

File: class11/python/AP1110-CalculoSalarioLiquido.py
# CalculoSalarioLiquido — Aula 11
# Entrada → Processamento → Saída: salário com descontos

horas = float(input("Horas trabalhadas: "))
valor_hora = float(input("Valor por hora: R$ "))
bruto = horas * valor_hora
inss = bruto * 0.08
liquido = bruto - inss
print(f"Salário Bruto:   R$ {bruto:.2f}")
print(f"Desconto INSS:   R$ {inss:.2f}")
print(f"Salário Líquido: R$ {liquido:.2f}")
